server_test.go

```diff
@@ -216,10 +222,14 @@ func TestServerStopFlag(t *testing.T) {
 func TestServerNewMessage(t *testing.T) {
 	t.Run("pushes new message into server.messages, returns this message", func(t *testing.T) {
 		server := &Server{messages: new(messages)}
-		message, messages := server.newMessage(), server.messages.items
+		message := server.newMessage()
+		server.messages.append(message)
 
+		server.messages.RLock()
+		messages := server.messages.items
 		assert.NotEmpty(t, messages)
```

Comment:
Here we are testing test-behavior from 226 line, not an implementation.

Reply:
I just removed this.